operational/flexera/custom_branding/README.md

```diff
@@ -0,0 +1,75 @@
+
+## Input Parameters
+
+This policy has the following input parameters required when launching the policy.
+
+### Applies to whole policy (Optional)
+
+- *Email addresses* - A list of email addresses to notify
+
+### Category : All Customer Settings
+
+- *Nav Logo Image* - The web address of the image resource file location to use in place of the Flexera logo that displays in the app navigation’s expanded state. Must start with http : // or https : //
+- *Nav Logo Small image* - The web address of the image resource file location to use in place of the small Flexera logo that displays in the app navigation’s collapsed state. Must start with http : // or https : //
+- *Marketing Footer Hide* - A boolean to determine show/hide of the marketing footer on the landing page.
+- *Marketing Submit Ideas Hide* - A boolean to determine show/hide of the Submit Ideas link on the landing page.
+- *Marketing Support Plan Hide* - A boolean to determine show/hide of the Support Plan Information link on the landing page.
+- *Marketing Open New Case Link* - A URL override pointing to your own Open New Case link on the landing page. Must start with http : // or https : //
+- *Marketing flexera com hide* - A boolean to determine show/hide of the Flexera.com link on the landing page.
+- *Marketing Support Help Link* - A URL override pointing to your own Support Help link on the landing page. Must start with http : // or https : //
+
+### Category : MSP Specific Settings
+
+- *Login Logo Image* - The web address of the image resource file location to use in place of the Flexera logo for unauthenticated pages like login, password reset, and so on. Must start with http  :// or https  ://
+- *Global Customer Support Link* - A URL override pointing to your own Customer Support link on the login page and elsewhere throughout the app. Must start with http : // or https : //
+- *Marketing Login Image* - The web address of the image resource file location to use in place of the Flexera marketing section on the login page. Must start with http : // or https : //
+- *Marketing Login CTA Link* - A URL override pointing to your own Download Now call to action link on the login page. Must start with http : // or https : //
+- *Marketing Login CTA text* - Plain text override of the Download Now call to action text displayed on the login page.
+- *Marketing Login Eyebrow Text* - Plain text to override of the eyebrow marketing text on the login page.
+- *Marketing Login Blurb Text* - Plain text to override of the marketing blurb on the login page.
+- *Marketing Login Title Text* - Plain text override of the marketing title text on the login page.
+
+The [Partner MSP Cusotmizations](https://docs.flexera.com/flexera/EN/Administration/PartnerMSPCustomizations.htm) page in the docs has detailed information on the settings and diagrams for custom branding
+
+## Policy Actions
+
+The following policy actions are taken when branding fails.
+
+- Send an email report
+- Create an incident

```

Comment:
Generally do not need to specify "create an incident" here. Look at other policy template READMEs for guidance on how things should be formatted.